stdlib/lang-parser/parserModule/readStream.bal

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
the file name convention is to use underscores, `read_stream.bal`. Please check all.

Reply:
Ballerina lang-parser implementation in no longer maintained and has already being removed from the ballerina-lang repo.